_posts/2017-08-25-introducing-sandboxfs.md

```diff
@@ -3,50 +3,52 @@ layout: posts
 title: Introducing sandboxfs
 ---
 
-sandboxfs is a new sandboxing technique for Bazel. It's experimental and
-subject to change, but it's available now for you to check out!
+sandboxfs is a new project to improve the way sandboxing works in Bazel by
+making it more efficient and correct.  It's experimental and subject to
+change, but it's available now for you to check out! Read on for details.
 
 # Correct builds
 
 As our motto claims, *correctness* is an integral part of Bazel. To achieve
 correctness, builds must be *hermetic* and
 [*reproducible*](https://reproducible-builds.org/), which means that all
-actions invoked by Bazel should be run in a well-defined environment. In other
-words: we want actions to run within a sandbox.
+actions invoked by Bazel should be run in a well-defined environment. In
+other words: we want actions to run within a sandbox.
 
 Let's consider an example. Think of a `cc_library` target that specifies
-`foo.cc` as a source. The `cc_library` rule contains a
-*compilation action* that turns the `foo.cc` source file into a `foo.o` object
-file. This action will run clang, and it will need to:
+`foo.cc` as a source. This `cc_library` target contains a *compilation
+action* that turns the `foo.cc` source file into a `foo.o` object file.
+This action will run clang, and it will need to:
 *  read `foo.cc`,
-*  read the transitive closure of all headers required by `foo.cc`
```

Comment:
Is it accurate to say "all files" here or something else specific? Reading transitive closures is a bit vague/jargon if we're still talking about files or something else specific. However, the suggestion below is also very wordy. Will the original "read transitive closures of headers" may make sense to your readers (you'll know this better than me)?

"read all files in the transitive closure of all headers required by foo.cc"

Reply:
What about just dropping "transitive closure" altogether? It's OK to not be too specific, as this is just an example.

"read all the headers required by ..."